nnvm/python/nnvm/to_relay.py

```diff
@@ -480,6 +480,7 @@ def to_relay(graph, shape_dict, dtype_dict, params):
                     "nnvm.to_relay: unsupported operator: {0}".format(op_name))
 
     outputs = [relay_map[nid] for nid in output_ids]
+    outputs = [x if not isinstance(x, expr.TupleWrapper) else x.astuple() for x in outputs]
```

Comment:
I think this is not enough.  Here is an example we also need to consider.
```python
import nnvm   
from nnvm.to_relay import to_relay

x = nnvm.sym.Variable("x")
y = nnvm.sym.split(x, indices_or_sections=2)
z = y[1]
graph = nnvm.graph.create(z)
print(graph.ir())
# Graph(%x) {
#   %1 = split(%x, indices_or_sections='2')
#   ret %1.1
# }

func, _ = to_relay(graph, {}, {}, {})
print(func)
# fn (%x: ) {
#   %0 = split(%x, indices_or_sections=int64(2))
#   %0  <= should be %0.1
# }  
```

In the case of TupleWrapper, I think we should return `x[index]` instead of `x.astuple()` where `index` can be taken from `json.loads(graph.json())['heads']`.

Reply:
Those cases are also handled now